cypress/e2e/tests/migration/dynamic-report/dependencies/sort.test.ts

```diff
@@ -0,0 +1,55 @@
+import { Analysis } from "../../../../models/migration/applicationinventory/analysis";
+import {
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+    validateSortBy,
+} from "../../../../../utils/utils";
+import { SEC } from "../../../../types/constants";
+import { Dependencies } from "../../../../models/migration/dynamic-report/dependencies/dependencies";
+
+describe(["@tier2"], "Dependencies sort validations", function () {
+    let application: Analysis;
+    const sortByList = ["Dependency name", "Labels", "Found in"];
+
+    before("Load data", function () {
+        login();
+
+        cy.intercept("GET", "/hub/application*").as("getApplication");
+    });
+
+    beforeEach("Test title", function () {
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
+        });
+    });
+
+    it("Creating data for sorting", function () {
+        application = new Analysis(
+            getRandomApplicationData("daytrader-app", {
+                sourceData: this.appData["daytrader-app"],
+            }),
+            getRandomAnalysisData(this.analysisData["source+dep_analysis_on_daytrader-app"])
+        );
+        application.create();
+        cy.wait("@getApplication");
+        application.analyze();
+        cy.wait(2 * SEC);
+        application.verifyAnalysisStatus("Completed");
+    });
```

Comment:
This shouldn't be a TC but be included in the before hook.

Reply:
And.... it will not work. I tried, but using fixtures doesn't work in before or before each sections, they can only be defined there

Reply:
It will work if you chain the promises (`.then`)